src/features/student/views/StudentSkillView/components/StudentSkillViewContainer/StudentSkillViewContainer.vue

```diff
@@ -0,0 +1,115 @@
+<script setup lang="ts">
+import type { SkillDetailedDTO } from '@/api/avenir-esr'
+import type { Component } from 'vue'
+import StudentSkillDetailedSection
+  from '@/features/student/views/StudentSkillView/components/StudentSkillDetailedSection/StudentSkillDetailedSection.vue'
+import StudentSkillEvaluateSection
+  from '@/features/student/views/StudentSkillView/components/StudentSkillEvaluateSection/StudentSkillEvaluateSection.vue'
+import StudentSkillLevelDetailedSection
+  from '@/features/student/views/StudentSkillView/components/StudentSkillLevelDetailedSection/StudentSkillLevelDetailedSection.vue'
+import StudentSkillProgressSection
+  from '@/features/student/views/StudentSkillView/components/StudentSkillProgressSection/StudentSkillProgressSection.vue'
+import { SkillItems } from '@/features/student/views/StudentSkillView/types'
+import { AvSideNavigation, type AvSideNavigationItem, MDI_ICONS } from '@avenirs-esr/avenirs-dsav'
+import { useI18n } from 'vue-i18n'
+
+const props = defineProps<{ skillDetailed: SkillDetailedDTO | undefined }>()
+const { skillDetailed } = toRefs(props)
```

Comment:
```suggestion
const { skillDetailed } = defineProps<{ skillDetailed: SkillDetailedDTO | undefined }>()
```
Pourquoi ne pas faire ça directement ?

Reply:
De base c'était pour que la props soit réactive, mais je n'ai pas essayé de la remettre comme ça après avoir ajouté la key sur le composant dans le parent